nix/tenzir/default.nix

```diff
@@ -316,6 +316,7 @@ let
               "-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=ON"
               "-DCPACK_GENERATOR=${if stdenv.hostPlatform.isDarwin then "TGZ;productbuild" else "TGZ;DEB;RPM"}"
               "-DTENZIR_UV_PATH:STRING=${lib.getExe uv-bin}"
+              "-DTENZIR_ALLOCATOR=mimalloc"
```

Comment:
@tobim Is this a good idea? On all other platforms, the mimalloc build lead to bad memory retention.